### DISCUSSION TITLE
shortcut disabling

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#2388] [Ideas]
Is there any way to disable certain shortcuts?
like I use ctrl w alot and I dont want to press ctrl q by accident
also what is the difference between ctrl w and just q?

**Comment by ghost** [comment]
I know its not formatted like an idea and more like a question but I think it is the idea for a good feature to be added

**Comment by kjk** [comment]
There is no way to disable shortcuts and there's no plan to allow such granularity of customization.

**Comment by GitHubRulesOK** [comment]
Perhaps now possible as @kjk has included in newer [Pre-release](https://www.sumatrapdfreader.org/prerelease) assignable keys thus try
```
Shortcuts [
	[
		Cmd = CmdNone
		Key = CTRL + q
	]
]
```
